include/date_time.h

```diff
@@ -62,13 +86,28 @@ int date_time_uptime_to_unix_time_ms(int64_t *uptime);
  */
 int date_time_now(int64_t *unix_time_ms);
 
+/** @brief Register an event handler for Date time library events.
+ *
+ *  @warning The library only allows for one event handler to be registered
+ *           at a time. De-register the previous handler before registering a
+ *           new.
```

Comment:
Is de-registration required? It looks like it's overwritten in the library even if it's not de-registered first.
It would also not be possible for a module to de-register a handler registered by another module, as it's typically a static function not known outside of its compile unit.

Reply:
@jtguggedal Yes, the handler is overwritten. I can change the "De-reg..." sentence to something in the likes of. "A passed in event handler in this function will overwrite the previously set event handler." Is this sufficient?

Reply:
Yep, that should be okay.